### DISCUSSION TITLE
Show Vimeo video from text link

### DISCUSSION THREAD

**Discussion by PerfectCamp** [henrygd/bigger-picture#31] [Q&A] [answered]
I'm trying to show a vimeo video after clicking a text link.  I have the latest version of your plugin.

`<a href="https://player.vimeo.com/video/739479052?autoplay=1&badge=0&autopause=0&player_id=0&app_id=58479"
	data-iframe="https://player.vimeo.com/video/739479052?autoplay=1&badge=0&autopause=0&player_id=0&app_id=58479" 
	data-caption="PerfectCamp Availability Map Search"
	class="lightbox-link"
	>Availability Map Search</a>`

My JS code:

`var bp = BiggerPicture({ target: document.body });
var videoLinks = document.querySelectorAll('.comparison-section .lightbox-link');
$('.comparison-section .lightbox-link').on('click', function(e) {
	e.preventDefault();
	bp.open({
		items: videoLinks,
		el: e.currentTarget,
	});
});`

Everything is working visually however in the console there is a 404 error upon clicking the link.

![image](https://github.com/henrygd/bigger-picture/assets/141071076/70ee8a3e-e0dc-4204-b19a-c43530671c58)

Does anyone know what could be the issue?

Thanks!

**Comment by henrygd** [comment, marked as answer]
I think the library is trying to load a thumbnail (`data-thumb`) but no thumbnail is defined.

The thumbnail is used as a placeholder before the media fully loads. Ideally this is something already loaded on the page so that it can be displayed instantly. See the [demo site](https://biggerpicture.henrygd.me/) for reference. If you're fine with the black background then you don't need to add one.

This is probably something I can add a check for at some point to avoid the network request.

**Reply by PerfectCamp** [reply]
Thanks for the quick reply.  I could probably just add a thumbnail image as a data attribute to solve the problem.  Great plugin!
